src/libsystemd-network/sd-ipv4ll.c

```diff
@@ -74,7 +74,7 @@ static sd_ipv4ll *ipv4ll_free(sd_ipv4ll *ll) {
 
 DEFINE_TRIVIAL_REF_UNREF_FUNC(sd_ipv4ll, sd_ipv4ll, ipv4ll_free);
 
-int sd_ipv4ll_new(sd_ipv4ll **ret) {
+int sd_ipv4ll_new(sd_ipv4ll **ret, uint64_t timeout_usec) {
```

Comment:
Rather than introducing another argument, please introduce a simple wrapper for sd_ipv4acd_set_timeout().